server/embed/etcd.go

```diff
@@ -217,7 +217,7 @@ func StartEtcd(inCfg *Config) (e *Etcd, err error) {
 		WatchProgressNotifyInterval:              cfg.ExperimentalWatchProgressNotifyInterval,
 		DowngradeCheckTime:                       cfg.ExperimentalDowngradeCheckTime,
 		WarningApplyDuration:                     cfg.ExperimentalWarningApplyDuration,
-		WarningUnaryRequestDuration:              cfg.ExperimentalWarningUnaryRequestDuration,
+		WarningUnaryRequestDuration:              cfg.WarningUnaryRequestDuration,
```

Comment:
If users only set `ExperimentalWarningUnaryRequestDuration`, we should continue to respect it, but print a warning log. cc @spzala @serathius

Reply:
sounds good, I'll add warning log about flag being depreciated. FYI, '--experimental ...' version of the flag is broken at the moment. I'm going to push another update